homeassistant/components/sql/sensor.py

```diff
@@ -137,6 +142,11 @@ def extra_state_attributes(self):
         """Return the state attributes."""
         return self._attributes
 
+    @property
```

Comment:
I believe you don't need to define the property (it will take the state_class from `_attr_state_class`

Reply:
Ah yes, true will fix that.
also apparently my git hook is gone ski have to fix that too